### PR TITLE
README.md change proposals

### PR DESCRIPTION
**Pull request by unsaved**
As you can see, I added a work-around to the Archlinux kernel build section, inserted a Bootloader setup section, and added steps that were essential to my installation to the sound and touchpad sections.

### CODE REVIEW

**Comment by idvoretskyi** [comment]
@unsaved, Have you reviewed the recommendations after the 4.7.4 kernel release?

File: README.md [hunk 1]
```diff
@@ -49,12 +51,35 @@ $ sudo make install
...
 ## Post-install steps
 
 Once installed reboot and load the kernel.
 
 ### Sound
 
+##### Enabling sound
+
+If not already present, install package `alsa-utils`.
+
```
Comment (raphael) [inline README.md]: Would you mind moving the below to a "Troubleshooting" section?

Comment (idvoretskyi) [inline README.md]: +1 to this proposal.

**Comment by raphael** [comment]
Going to close this as some of the info in there is now stale.

**Pull request closed without merging**